src/EventStore.ClientAPI.Embedded/EmbeddedResponders.cs

```diff
@@ -57,6 +57,61 @@ protected override WriteResult TransformResponse(ClientMessage.WriteEventsComple
             }
         }
 
+        internal class ConditionalAppendToStream :
+            EmbeddedResponderBase<ConditionalWriteResult, ClientMessage.WriteEventsCompleted>
+        {
+            private readonly int _expectedVersion;
+            private readonly string _stream;
+
+            public ConditionalAppendToStream(TaskCompletionSource<ConditionalWriteResult> source, string stream, int expectedVersion)
+                : base(source)
+            {
+                _stream = stream;
+                _expectedVersion = expectedVersion;
```

Comment:
Assigned a value but never used. Currently failing on travis because it treats warnings as errors.